docs/user/contributing.rst

```diff
@@ -157,3 +157,15 @@ TorchGeo has a number of tutorials included in the documentation that can be run
 .. code-block:: console
 
    $ pytest --nbmake docs/tutorials
+
+
+Datasets
+--------
+
+A major component of TorchGeo is the large collection of :mod:`torchgeo.datasets` that have been implemented. Adding new datasets to this list is a great way to contribute to the library. A brief checklist to follow when implementing a new dataset:
+
+* Implement the dataset extending either :class:`GeoDataset <torchgeo.datasets.GeoDataset>` or :class:`VisionDataset <torchgeo.datasets.GeoDataset>`
```

Comment:
```suggestion
* Implement the dataset extending either :class:`~torchgeo.datasets.GeoDataset` or :class:`~torchgeo.datasets.GeoDataset`
```
See if the tilde has the same affect (link works, and only shows class name, not full path)

Reply:
Somewhere we should add guidance for how to decide between these. Doesn't have to be here or now though